src/frontend/components/Results/Results.css

```diff
@@ -0,0 +1,20 @@
+.resultsTable td, .resultsTable th{
+    color: white;
+    border: 1px solid white;
+    padding: 20px;
+}
+
+.user-image {
+    height: 100px;
+    width: 100px;
+    padding: 20px;
+}
+
+.resultsTable {
+    /* transform: translate(20%, 20%); */
```

Comment:
comment can be removed

Reply:
I have left the comments on purpose so that Kainy can use the code later